include/params_py.h

```diff
@@ -736,6 +759,37 @@ const String paramBmsToInverter PROGMEM = "["
     "'max':3600"
   "}," 
 
+  "{"
+    "'label':'Überschreibe SOC',"
+    "'type':"+String(HTML_SEPARATION)+","
+    "'help':'Berechnet den SOC linear anhand der Ø Zellspannung und überschreibt den von BMS gemeldeten. Für JKBMS V1.3 notwendig.'"
+  "},"
+  "{"
+    "'name':"+String(ID_PARAM_INVERTER_OVERWRITE_SOC_EN)+","
+    "'label':'Ein/Aus',"
+    "'type':"+String(HTML_INPUTCHECKBOX)+","
+    "'default':'0'"
+  "},"
+  "{"
+    "'name':"+String(ID_PARAM_INVERTER_OVERWRITE_SOC_MIN_VOLTAGE)+","
+    "'label':'Min. Spannung',"
+    "'unit':'mV',"
+    "'type':"+String(HTML_INPUTNUMBER)+","
+    "'default':'2000',"
+    "'min':0,"
+    "'max':3600"
+  "}," 
+  "{"
+    "'name':"+String(ID_PARAM_INVERTER_OVERWRITE_SOC_MAX_VOLTAGE)+","
+    "'label':'Max. Spannung',"
+    "'unit':'mV',"
+    "'type':"+String(HTML_INPUTNUMBER)+","
+    "'default':'3600',"
+    "'min':0,"
+    "'max':3600"
+  "},"   
+
+
```

Comment:
Ich würde die Parameter „Überschreibe SOC“ in die seriellen Schnittstelleneinstellungen machen (`paramSerial`; dort mit in die vorhandene group). Dort kann dies dann für jede Schnittstelle eingestellt werden. Die Berechnung des SoCs würde ich dann nach dem Holen der Daten vom BMS in der `BscSerial::cyclicRun()` machen. Damit steht dann in den BMS Daten der „richtige“ SoC. Somit kann an jeder anderen Stelle auch der SoC gelesen werden (z.B. für das Display, oder wenn es jemand nur über MQTT verteilen möchte).

Reply:
ja macht sinn